src/map-interactions.react.js

```diff
@@ -325,7 +326,9 @@ export default class Interactions extends Component {
   render() {
     return (
       <div
-        ref="container"
+        ref={container => {
```

Comment:
It's the recommended pattern by FB: https://facebook.github.io/react/docs/refs-and-the-dom.html 

According to those docs, it should only be called after mount and before unmounting. I'm not sure if our issue in v3 was because we're using `createElement` directly vs. going through jsx.